BizStream/brain-rot-erator
# Feature/brain 6/8 restructure & download/delete selection UI

- BackgroundScheduler to cleanup temp clips after an hour
- Python endpoints to send links (more efficient than encodings) to clips back to Next and delete downloaded clips
- Deleted unnecessary /api route folder because I'm just making the calls directly in the client component
- Added env vars to hide paths to Python api on Next's side
- Added warning popup about scheduled cleanup
- Fixed the way clips were being downloaded because Python is sending links now
- Created a call to delete a clip that has been downloaded and removed that clip from videos array upon succession to avoid hydration issues
- Left error-handling/comments in for testing of download to delete functionality
- Known bugs: after deleting/downloading a clip, warning pops up on home page for 1/2 a second, also making more Python api calls than necessary to display/play each video. Possibly deleting/downloading the wrong clip at times (will need to be tested)

QA:
1. Fill out fields
2. Submit form
3. Dismiss popups (shows 2 in dev mode)
4. Select a clip and download it
5. Select multiple clips and download
6. Click the return button to go to the home page
7. Mess around with the browser navigation
8. Redo this process but make sure to select and download the first two clips
